catroid/src/main/AndroidManifest.xml

```diff
@@ -124,34 +126,28 @@
                 <category android:name="android.intent.category.BROWSABLE" />
                 <data
                     android:host="*"
+                    android:mimeType="application/octet-stream"
```

Comment:
application/zip  would be more accurate

Reply:
application/zip does not work on my Android 7.0 device